cumulusci.yml

```diff
@@ -9,7 +9,7 @@ project:
         install_class: STG_InstallScript
         uninstall_class: STG_UninstallScript
     git:
-        repo_url: https://github.com/SalesforceFoundation/Cumulus
+        repo_url: https://github.com/SalesforceFoundation/NPSP
```

Comment:
This change needs to be coordinated with updating the repo URL in MetaDeploy, or else it'll break publishing new versions of the install plans.